internal/app/admin/http/handler.go

```diff
@@ -23,6 +24,10 @@ type Handler struct {
 	redirect    bool
 }
 
+const logUrl = "/log_level"
```

Comment:
nit: may be use a block to define constants https://github.com/envoyproxy/xds-relay/blob/master/internal/app/metrics/metrics.go#L11-L15